tests/HUBViewControllerTests.m

```diff
@@ -1863,6 +1864,7 @@ - (void)viewController:(UIViewController<HUBViewController> *)viewController
       willAppearInView:(UIView *)componentView
 {
     XCTAssertEqual(viewController, self.viewController);
+    XCTAssertFalse([componentView isKindOfClass:[HUBComponentCollectionViewCell class]]);
```

Comment:
Maybe add an assertion to make sure you get the correct view back, not only "not a collection view cell"?